packages/1155-contracts/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @zoralabs/zora-1155-contracts
 
+## 2.1.1-premint-api.0
```

Comment:
before merging back into main:

`yarn changeset pre exit`

in general the `prerelease` branch should be a separate branch from the active dev work, so that prerelease mode and changelogs don't get merged into main

i.e. you have one branch called `premint-api-helper` and another one called `premint-api-helper-prerelease` which is a stacked diff on top of `premint-api-helper`

Reply:
Opted to delete all pre-release changes and write a patch change instead.